commandline/WSL/faq.md

```diff
@@ -86,32 +57,8 @@ WSL does **not** aim to support GUI desktops or applications (e.g. Gnome, KDE, e
 
 Also, even though you will be able to run many popular server applications (e.g. Redis), we do not recommend WSL for server scenarios – Microsoft offers a variety of solutions for running production Ubuntu workloads in Azure, Hyper-V, and Docker. 
 
-### Why is this a beta release?
-This is the first release of a brand new technology. There will be gaps. We know some of them, and we’re certain you will find many more.
-Again, this is a beta release – expect things to fail. Some tools will crash and/or not run. But please be sure to let us know when you run into issues – we’re working hard to fix problems and dramatically improve WSL over time.
-
-### How do I update Bash on Ubuntu on Windows?

```

Comment:
we should keep this section don't you think?  Updating is still best via the distro specific upgrade mechanism vs. store updates and we should explain that.